Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/SolenoidReverse.java

```diff
@@ -9,11 +9,11 @@
  */
 public class SolenoidReverse extends Command {
 
-	boolean isActive;
+	boolean isFinished;
 	
     public SolenoidReverse() {
     	requires(Robot.solenoid);
-    	isActive = true;
+    	isFinished = true;
```

Comment:
We need to look at how we're using `isFinished `across the different commands. `isFinished` should indicated by either a flag from the subsystem or some other way. The way that commands work is that they get put into a schedule. This schedule runs every 20ms while the match is running. If we simply put `isFinished = true;` some subsystem call `isFinished = false;` we may end up stopping what is happening for the command. In this particular case there is nothing that is really telling the scheduler the command is or is not executing. This could result in undesired behavior like the the solenoid extending and retracting in the same time period (i.e. a button press) vs extending and then waiting for another button press or button release to retract.